merge_requests_test.go

```diff
@@ -126,6 +127,34 @@ func TestGetMergeRequest(t *testing.T) {
 	require.Equal(t, mergeRequest.HasConflicts, true)
 }
 
+func TestGetListMergeRequestWithMergeStatusRecheck(t *testing.T) {
```

Comment:
Not sure if this test adds much value. Maybe just set the field in the `TestListProjectMergeRequests` test instead?